app/src/main/java/com/bebetterprogrammer/tictactoe/activities/Whowin.kt

```diff
@@ -0,0 +1,64 @@
+package com.bebetterprogrammer.tictactoe.activities
+
+import com.bebetterprogrammer.tictactoe.R
+import kotlinx.android.synthetic.main.activity_gameplay.*
+
+class Whowin {
```

Comment:
This class will not be needed as I have commented on `Iswin` class. By those changes, that class `Iswin` class can be used generically. As you are doing the almost same computation here.